consensus/core/src/commit_syncer.rs

```diff
@@ -302,7 +308,15 @@ impl<C: NetworkClient> CommitSyncer<C> {
                     return (commit_range.end(), commits, blocks);
                 }
                 Err(e) => {
-                    warn!("Failed to fetch: {}", e);
+                    warn!("Failed to fetch: {}", e.to_string());
```

Comment:
Is `to_string()` necessary?

Reply:
ah should have cleaned this up - will revert